tests/Feature/UserRepositoryTest.php

```diff
@@ -721,4 +721,26 @@ public function testGetByEmailsCachingTTL()
         // Verify now a total of two api calls was made.
         self::assertCount(2, $this->guzzleContainer);
     }
+
+    /**
+     * Verifies that deletion works correctly
+     */
+    public function testDeleteOne()
+    {
+        // Altered response from Auth0 API documentation
+        $this->mockedResponses = [new Response(200, [], '')];
```

Comment:
```suggestion
        $this->mockedResponses = [new Response(204, [], '')];
```
According to Auth0 docs the API returns 204 when the user was successfully deleted: https://auth0.com/docs/api/management/v2#!/Users/delete_users_by_id